mtp_api/apps/credit/managers.py

```diff
@@ -0,0 +1,82 @@
+from django.db import models, connection
+
+from .constants import LOG_ACTIONS, CREDIT_STATUS
+
+
+class CreditQuerySet(models.QuerySet):
+
+    def available(self):
+        return self.filter(**self.model.STATUS_LOOKUP[CREDIT_STATUS.AVAILABLE])
+
+    def locked(self):
+        return self.filter(**self.model.STATUS_LOOKUP[CREDIT_STATUS.LOCKED])
+
+    def credited(self):
+        return self.filter(**self.model.STATUS_LOOKUP[CREDIT_STATUS.CREDITED])
+
+    def refunded(self):
+        return self.filter(**self.model.STATUS_LOOKUP[CREDIT_STATUS.REFUNDED])
+
+    def refund_pending(self):
+        return self.filter(**self.model.STATUS_LOOKUP[CREDIT_STATUS.REFUND_PENDING])
+
+    def locked_by(self, user):
+        return self.filter(owner=user)
+
+    def update_prisons(self):
```

Comment:
this doesn't really feel like a query set method because it isn't compose-able with others, perhaps makes sense as a model manager method or even model class method? not too bothered tho